third_party/move/move-vm/runtime/src/debug.rs

```diff
@@ -146,7 +145,7 @@ impl DebugContext {
         interpreter: &dyn InterpreterDebugInterface,
     ) {
         let instr_string = format!("{:?}", instr);
-        let function_string = function.name_as_pretty_string();
+        let function_string = format!("{}::{}", function.name_as_pretty_string(), pc);
```

Comment:
Added pc to function string so that breakpoint can be set using function name and pc combination.